Sem. 06/Pract.06/Task09.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+constexpr int MAX_SIZE = 100;
+
+void gettingTheNumbersIn(int arr[], int n) {
+    for (int i = 0; i < n; i++) {
+        std::cin >> arr[i];
+    }
+}
+
+void sortingTheArray(int arr[], int n) {
```

Comment:
In general, you do not need to sort the array to solve the problem. But if you want to sort it using bubble sort (the way you are doing it here), you can read[ this article](https://www.geeksforgeeks.org/bubble-sort-algorithm/) to optimize it :)